usage/help.go

```diff
@@ -30,6 +30,10 @@ func HelpCommand() cli.Command {
 				Name:  "html",
 				Usage: "The export <directory> for HTML docs.",
 			},
+			cli.StringFlag{
+				Name:  "report",
+				Usage: "The report <directory> for JSON report based on HTML docs.",
+			},
```

Comment:
The way that the code works, looks like this should be a `cli.BoolFlag` instead of a string with the directory, with this, we will generate the report with `step help --html <directory> --report`

Reply:
OK. That works for me too. I'll make the changes.